templates/devspace.yaml.tpl

```diff
@@ -310,10 +310,17 @@ profiles:
       - op: add
         path: hooks
         value:
-          name: reset-dev
+          name: reset-dev-interrupt
           events: ["devCommand:interrupt"]
           command: |-
             "${DEVENV_DEVSPACE_BIN}" reset pods -s
+      - op: add
+        path: hooks
+        value:
+          name: reset-dev-error
+          events: ["error:sync:app"]
```

Comment:
when does this event occur ?

Reply:
This event occurs when the e2e test fails. I want to make sure that we clean everything up so that it's a fresh run next time. 

Kuberenets throws errors about multiple pods using mounting the same secrets if we don't clean it up properly.